src/routes/(console)/project-[region]-[project]/auth/createTeam.svelte

```diff
@@ -10,12 +10,14 @@
     import { Icon, Tag } from '@appwrite.io/pink-svelte';
     import { createEventDispatcher } from 'svelte';
 
-    export let showCreate = false;
+    let { showCreate = $bindable(false) }: { showCreate: boolean } = $props();
 
     const dispatch = createEventDispatcher();
 
-    let name: string, id: string, error: string;
-    let showCustomId = false;
+    let name = $state<string>('');
```

Comment:
```suggestion
    let name = $state('');
```
when its the only type, no need to specify it explicitly, its inferred from the value. similar to how `showCustomId` is below.